### ISSUE TITLE
Error when building setup.py

### ISSUE THREAD

**Issue by Ghalif** [mapillary/OpenSfM#920]
Hi. I have been followed all the instructions for Windows guide and installed all the dependencies successfully. However I am getting this error when I attempt to build setup.py. Can someone help me with resolving this error? Thank you.

```
PS C:\src\OpenSfM> python setup.py build
Configuring for python 3.9...
Traceback (most recent call last):
  File "C:\src\OpenSfM\setup.py", line 59, in <module>
    configure_c_extension()
  File "C:\src\OpenSfM\setup.py", line 43, in configure_c_extension
    subprocess.check_call(cmake_command, cwd="cmake_build")
  File "C:\Users\ismai\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 368, in check_call
    retcode = call(*popenargs, **kwargs)
  File "C:\Users\ismai\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Users\ismai\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\ismai\AppData\Local\Programs\Python\Python39\lib\subprocess.py", line 1420, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

_Side Note: In case you are wondering why I am using 3.9.7 version of python. It is because when I try to install the dependencies using 3.10 python, I am getting error when it is trying to install opencv-python==4.5.1.48._

**Comment by fabianschenk** [comment]
Hi @Ghalif ,
I think, you're missing a library `pdftk`. A quick Google search showed [this](https://stackoverflow.com/questions/69039199/pythonpdf-filenotfounderror-winerror-2-the-system-cannot-find-the-file-speci).

Feel free to create a PR with updated `requirements.txt`.

Best,
Fabian

**Comment by Ghalif** [comment]
I don't think that is the missing library cx the error is not related to any pdf.
Anyways, for some unknown reason, it is working now and has been built. I think maybe I was missing a CMake file. Cx after I got this error, I tried to build several other SfM software (Theia and OpenMVG) and in the process installed a bunch of dependences (including CMake). So don't know what to pin-point as the solution.
